src/views/AddPriceMultiple.vue

```diff
@@ -232,7 +232,12 @@
                 min="0"
                 hide-details="auto"
                 :suffix="productPriceForm.currency"
-              ></v-text-field>
+                >
+                  <template v-slot:prepend>
+                    <!-- image from https://www.svgrepo.com/svg/32717/currency-exchange -->
+                    <img src="/currency-exchange-svgrepo-com.svg" class="icon-info-currency" @click="changeCurrencyDialog = true" />
```

Comment:
i guess there's no similar mdi icon :(

Reply:
i would add a `title` (can be done in a seperate PR and add accessibility elsewhere)